.github/workflows/test.yml

```
@@ -37,9 +51,10 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install --upgrade tox tox-gh-actions
 
-    - name: Tox tests
-      run: |
-        tox -v
+    - name: Unit tests for py${{ matrix.python-version }}-django${{ matrix.django-version }}
```

1. It's not relevant if those are  "unit tests" or any other type of tests
2. I want to prevent the GitHub config to have to be synced with the tox config as much as possible, so let's not use the env specifiers in descriptive text

```suggestion
    - name: Tox tests (Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }})
```

---

I don't really care too much, but strictly speaking ... what are Tox tests at all? Tox is the test runner.

Granted, that's nit-picking.